app/views/session_timeout/_warning.html.erb

```diff
@@ -38,7 +38,7 @@
     </p>
   </div>
   <%= render ButtonComponent.new(
-        type: :button,
+        url: new_user_session_url(timeout: :session, request_id: sp_session[:request_id]),
```

Comment:
This is intentionally identical to the URL used for the _fully-authenticated_ session expiration redirect URL:

https://github.com/18F/identity-idp/blob/621346b30ba5efc7065a3b394521e92914e8ff98/app/views/session_timeout/_ping.html.erb#L3

The thought being that we could get rid of `session-timeout-ping.ts` in favor of `session-expire-session.ts`, and it would "just work" the same way.

Reply:
The only gotcha here is that including `?timeout=session` means that a user will see the session timeout alert if they click "Continue sign in" after their session is fully expired. I think it's a reasonable and arguably-expected notice to be shown.

![image](https://github.com/user-attachments/assets/52fb50a3-0a90-4eb4-a8cc-69071b85123c)